lib/eco/autolboxdetector.py

```diff
@@ -383,6 +390,8 @@ def next_token(self):
             t = self.tokeniter()
             self.last_read = t[3][-1]
             self.last_token_value = t[0]
+            length = sum([len(x.symbol.name) for x in t[3]])
```

Comment:
If this is in a performance critical bit of code, `sum(map(...))` might be faster (if it works)?

Reply:
Not super critical, but won't hurt.

Reply:
Nevermind, I just noticed the variable isn't even used. So I can just delete the whole thing.